components/script/dom/bluetoothremotegattcharacteristic.rs

```diff
@@ -150,4 +150,21 @@ impl BluetoothRemoteGATTCharacteristicMethods for BluetoothRemoteGATTCharacteris
             Ok(self.GetValue().unwrap())
         }
     }
+
+    // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothremotegattcharacteristic-writevalue
+    fn WriteValue(&self, value: Vec<u8>) {
+        let (sender, receiver) = ipc::channel().unwrap();
+        self.get_bluetooth_thread().send(
+            BluetoothMethodMsg::WriteValue(self.get_instance_id(), value, sender)).unwrap();
```

Comment:
I used this for consistency, but there will be a style refactor patch where your solution will be used.